src/Avalonia.Controls/Primitives/AccessText.cs

```diff
@@ -111,8 +111,13 @@ protected override void OnDetachedFromVisualTree(VisualTreeAttachmentEventArgs e
         /// </summary>
         /// <param name="text">The text.</param>
         /// <returns>The text with the first underscore stripped.</returns>
-        private string StripAccessKey(string text)
+        private string? StripAccessKey(string? text)
```

Comment:
[return: NotNullWhenNotNull] here?

Reply:
Though probably not worth it

Reply:
Might as well I guess, though yeah it's rather academic.